modules/rhpam-apb/roles/deploy-amq/tasks/route.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "[AMQ-JOLOKIA-CONSOLE] set http route state to {{ state }}"
+  k8s:
+    state: '{{ state }}'
+    definition:
+      apiVersion: v1
+      kind: Route
+      metadata:
+        name: amq-jolokia-console
+        namespace: '{{ namespace }}'
+        labels:
+          app: '{{ kieserver_deployment_name }}'
+          application: '{{ application_name }}'
+          service: '{{ application_name_amq }}'
+      spec:
+        port:
+          targetPort: amq-jolokia
+        to:
+          kind: Service
+          name: '{{ amq_service_name }}'
+  register: route_raw_amq
+
+- name: "[AMQ-JOLOKIA-CONSOLE] set route host"
+  set_fact:
+    sr_host: "{{ route_raw_amq.result.spec.host }}"
```

Comment:
Where is this used?

Reply:
This route will expose the AMQ console.

Reply:
I see you are creating the route but why you need to save create the fact `sr_host` with the host?

Reply:
Ah I see your point, yes, not needed.